etcdmain/config.go

```diff
@@ -279,15 +277,14 @@ func (cfg *config) Parse(arguments []string) error {
 		os.Exit(2)
 	}
 	if len(cfg.FlagSet.Args()) != 0 {
+		if cfg.FlagSet.Arg(0) == "version" {
```

Comment:
this is a wrong place to do the sub command thing (this needs to be done in main.go as other sub commands). also i think probably etcd --version is good enough for now.